tests/acceptance/features-test.js

```diff
@@ -13,6 +15,16 @@ module('Acceptance: Features', {
   }
 });
 
+test('features are defined in app config', function() {
+  ok(features.enabled('feature-from-config'), 'feature-from-config is enabled automatically');
```

Comment:
Yep, I added DOM-based assertions for this as well, and added to the initializer test too.